tests/conftest.py

```diff
@@ -1,16 +1,13 @@
+from __future__ import annotations
+
 import sys
-from typing import Any, Dict, Tuple
+from typing import Any
 
 import pytest
 
 
 @pytest.fixture
-def os_and_version() -> str:
-    return sys.platform + "_" + ".".join(map(str, sys.version_info[:2]))
-
-
-@pytest.fixture
-def anyio_backend() -> Tuple[str, Dict[str, Any]]:
+def anyio_backend() -> tuple[str, dict[str, Any]]:
     if sys.platform == "win32":
         from asyncio.windows_events import WindowsSelectorEventLoopPolicy
```

Comment:
Maybe we should do something similar to this in the regular lib as well. We get a lot of issues from Windows users due to this.